core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -64,171 +66,210 @@ THE SOFTWARE.
+                                ${p.backupVersion}
+                              </span>
+                            </button>
+                          </form>
+                        </j:if>
+                      </div>
+                    </td>
+                    <td class="jenkins-table__cell--tight">
+                      <div class="jenkins-table__cell__button-wrapper">
+                        <j:choose>
+                          <j:when test="${p.hasMandatoryDependents()}">
+                            <j:set var="uninstallPossible" value="true"/>
+                          </j:when>
+                          <j:otherwise>
+                            <j:set var="uninstallPossible" value="false"/>
+                          </j:otherwise>
+                        </j:choose>
+                        <j:choose>
+                          <j:when test="${p.isDeleted()}">
+                            <p>${%Uninstallation pending}</p>
+                          </j:when>
+                          <j:otherwise>
+                            <j:if test="${p.hasMandatoryDependents() or p.hasImpliedDependents()}">
+                              <div class="dependent-list">
+                                <j:forEach var="dependent" items="${p.mandatoryDependents}">
+                                  <span data-plugin-id="${dependent}"></span>
+                                </j:forEach>
+                                <j:set var="impliedDependents" value="${p.impliedDependents}"/>
+                                <j:choose>
+                                  <j:when test="${impliedDependents.size() lt 15}">
+                                    <j:forEach var="dependent" items="${impliedDependents}">
+                                      <span data-plugin-id="${dependent}"></span>
+                                    </j:forEach>
+                                  </j:when>
+                                  <j:otherwise>
+                                    <p>
+                                      ${%detached-many-dependents(impliedDependents.size())}
+                                    </p>
+                                  </j:otherwise>
+                                </j:choose>
+                              </div>
+                            </j:if>
+                            <j:if test="${p.hasMandatoryDependencies()}">
+                              <div class="dependency-list">
+                                <j:forEach var="dependency" items="${p.mandatoryDependencies}">
+                                  <span data-plugin-id="${dependency.shortName}"></span>
+                                </j:forEach>
+                              </div>
+                            </j:if>
+                            <form class="jenkins-buttons-row" method="post" action="plugin/${p.shortName}/uninstall">
```

Comment:
🚫 Blocker: You can now uninstall plugins that are depended on as mandatory dependeices

I can uninstall the credentials plugin which shouldn't be possible